src/jobs/build_push_image.yml

```diff
@@ -86,4 +90,5 @@ steps:
   - steps: << parameters.prebuild_steps >>
   - build_push:
       config: << parameters.path >>/isopod.yml
+      work_dir: << parameters.work_dir>>
```

Comment:
i presume workspace attachment on `.` still works? otherwise you have to review L88